app/views/newMain.scala.html

```diff
@@ -0,0 +1,39 @@
+@import assets.AssetsResolver
+@import views.ViewHelpers._
+@import switchboard.Switches
+@(
+	title: String,
+	description: Option[String] = None,
+)(implicit assets: AssetsResolver, request: RequestHeader, switches: Switches)
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
```

Comment:
since this page isn't linked to it shouldn't be reachable by web crawlers, but do we want a noindex meta tag to prevent it showing up in search indexes?